spec/01-unit/03-conf_loader_spec.lua

```diff
@@ -2686,4 +2686,70 @@ describe("Configuration loader", function()
     end)
   end)
 
+  describe("pluginserver config", function()
+    describe("fails if", function()
+      it("no query_command and not found in default location", function()
+        local _, err = conf_loader(nil, {
+          pluginserver_names = "gopher",
+          -- query_command = {},
+        })
+        assert.is_not_nil(err)
+        assert.matches("query_command undefined for pluginserver gopher", err)
+      end)
+
+      -- TODO need one more test case:
```

Comment:
Was this TODO missed or will this be addressed in a future PR?

Reply:
This is related to your comment below; I have moved the comment to the appropriate location.